x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4

```diff
@@ -22,28 +34,30 @@ query
 sourceCommand
     : explainCommand
     | fromCommand
+    | metaCommand
     | rowCommand
-    | metricsCommand
     | showCommand
-    | metaCommand
+    // in development
+    | {devVersion()}? metricsCommand
     ;
 
 processingCommand
     : evalCommand
-    | inlinestatsCommand
-    | limitCommand
-    | lookupCommand
+    | whereCommand
     | keepCommand
-    | sortCommand
+    | limitCommand
     | statsCommand
-    | whereCommand
+    | sortCommand
     | dropCommand
     | renameCommand
     | dissectCommand
     | grokCommand
     | enrichCommand
     | mvExpandCommand
-    | matchCommand
+    // in development
+    | {devVersion()}? inlinestatsCommand
+    | {devVersion()}? lookupCommand
+    | {devVersion()}? matchCommand
```

Comment:
gate also the in-development commands.

Reply:
Why do these need to be gated in the lexer _and_ parser? Wouldn't one of the two suffice?